tests/Feature/CommandFixPermissionsTest.php

```diff
@@ -50,8 +50,8 @@ public function testFixPermissions(): void
 		])->assertSuccessful();
 
 		clearstatcache(true);
-		static::assertEquals(00664, fileperms($filePath) & 07777);
-		static::assertEquals(02775, fileperms($dirPath) & 07777);
+		static::assertEquals(00664, fileperms($filePath) & 07777, $filePath);
+		static::assertEquals(02775, fileperms($dirPath) & 07777, $dirPath);
```

Comment:
Why this change? I am not sure if it even improves anything. If the 3rd parameter is not given, the error message in case of a failure defaults to `"Failed asserting that (2nd parameter) equals (value of 1st parameter)"`. While I agree that the error message could be improved and made more specific, the error message now is _only_ the path of the problematic file/directory without any indication what is even wrong. I wouldn't call this an improvement.

Reply:
I assume this has been fixed in https://github.com/LycheeOrg/Lychee/pull/1469/commits/fae68b46c3e00709a6f54df5e1f9a284c14fe7d1, right?

Reply:
yes.